src/actions.js

```diff
@@ -9,69 +7,43 @@ export const REGISTER_USER_ERROR = "REGISTER_USER_ERROR";
 export const LOGIN_USER = "LOGIN_USER";
 export const LOGIN_USER_SUCCESS = "LOGIN_USER_SUCCESS";
 export const LOGIN_USER_ERROR = "LOGIN_USER_ERROR";
-const registerUserSuccess = (data, status) => {
-	return {
-		type: REGISTER_USER_SUCCESS,
-		user: data,
-		status: status
-	};
+const registerUserSuccess = data => {
+  return {
+    type: REGISTER_USER_SUCCESS,
+    user: data
+  };
 };
 const loginUserSuccess = data => {
-	return {
-		type: LOGIN_USER_SUCCESS,
-		token: data
-	};
+  return {
+    type: LOGIN_USER_SUCCESS,
+    token: data
+  };
 };
 
 export const loginUserAction = user => {
-	// needs to dispatch, so it is first argument
-	return dispatch => {
-		var session_url = "http://localhost:8001/api/login";
-
-		let formData = new FormData(); //formdata object
-
-		formData.append("username", user.username); //append the values with key, value pair
-		formData.append("password", user.password);
-
-		const config = {
-			headers: { "content-type": "multipart/form-data" }
-		};
-		axios
-			.post(session_url, formData, config)
-			.then(response => {
-				setCookie("token", response.data['token'], 2);
-				// this.props.history.push('/dashboard');
-				dispatch(loginUserSuccess(response.data));
-			})
-			.catch(error => {
-				console.error(error);
-			});
-	};
+  return dispatch => {
+    const data = {
+      username: user.username,
+      password: user.password
+    };
+    login(data).then(response => {
+      localStorage.setItem("token", response.token);
```

Comment:
Why do we need to store the token in localStorage?

Reply:
because this way we can check token everywhere we want if we don't do this then we have to map token in every component which is not a good way this is easy and good way i think